packages/cli/lib/sandbox-create.js

```diff
@@ -0,0 +1,337 @@
+          )
+        );
+      } else {
+        const baseUrl = getHubSpotWebsiteOrigin(getValidEnv(getEnv(accountId)));
+        logger.error(
+          i18n(
+            `${i18nKey}.failure.limit.developer.${plural ? 'other' : 'one'}`,
+            {
+              accountName: accountConfig.name || accountId,
+              limit: devSandboxLimit,
+              link: `${baseUrl}/sandboxes-developer/${accountId}/development`,
+            }
+          )
+        );
+      }
+      logger.log('');
+    } else if (
+      isSpecifiedError(
+        err,
+        400,
+        'VALIDATION_ERROR',
+        'SandboxErrors.NUM_STANDARD_SANDBOXES_LIMIT_EXCEEDED_ERROR'
+      ) &&
+      err.error &&
+      err.error.message
+    ) {
+      logger.log('');
+      const standardSandboxLimit = getSandboxLimit(err.error);
+      const plural = standardSandboxLimit !== 1;
+      const hasStandardSandboxes = getHasSandboxesByType(
+        accountConfig,
+        STANDARD_SANDBOX
+      );
+      if (hasStandardSandboxes) {
+        logger.error(
+          i18n(
+            `${i18nKey}.failure.alreadyInConfig.standard.${
+              plural ? 'other' : 'one'
+            }`,
+            {
+              accountName: accountConfig.name || accountId,
+              limit: standardSandboxLimit,
+            }
+          )
+        );
+      } else {
+        const baseUrl = getHubSpotWebsiteOrigin(getValidEnv(getEnv(accountId)));
+        logger.error(
+          i18n(
+            `${i18nKey}.failure.limit.standard.${plural ? 'other' : 'one'}`,
+            {
+              accountName: accountConfig.name || accountId,
+              limit: standardSandboxLimit,
+              link: `${baseUrl}/sandboxes-developer/${accountId}/standard`,
+            }
+          )
+        );
+      }
+      logger.log('');
+    } else {
+      logErrorInstance(err);
+    }
+    throw err;
+  }
+
+  let sandboxConfigName;
+
+  try {
+    // Response contains PAK, save to config here
+    sandboxConfigName = await saveSandboxToConfig(env, result, force);
+  } catch (err) {
+    logErrorInstance(err);
+    throw err;
+  }
+
+  if (skipDefaultAccountPrompt || force) {
+    updateDefaultAccount(sandboxConfigName);
+  } else {
+    const setAsDefault = await setAsDefaultAccountPrompt(sandboxConfigName);
+    if (setAsDefault) {
+      logger.success(
+        i18n(`cli.lib.prompts.setAsDefaultAccountPrompt.setAsDefaultAccount`, {
+          accountName: sandboxConfigName,
+        })
+      );
+    } else {
+      const config = getConfig();
+      logger.info(
+        i18n(
+          `cli.lib.prompts.setAsDefaultAccountPrompt.keepingCurrentDefault`,
+          {
+            accountName: config.defaultPortal,
+          }
+        )
+      );
+    }
+  }
+
+  // If creating standard sandbox, prompt user to sync assets
+  if (allowSyncAssets) {
+    if (sandboxType === STANDARD_SANDBOX) {
+      const syncI18nKey = 'cli.commands.sandbox.subcommands.sync';
```

Comment:
Can this also move to the `cli.lib` section of the lang file now that it's not in the command file anymore?